src/fhirtypes/common.ts

```diff
@@ -326,8 +338,18 @@ export function applyMixinRules(
   fshDefinition: Profile | Extension | Instance,
   tank: FSHTank
 ): void {
+  if (fshDefinition.mixins.length > 0) {
+    const insertString = '* insert ' + fshDefinition.mixins.join('\n* insert ') + '\n';
```

Comment:
Wait, but the code you have is the case where there is no extra `\n` at the end, since `join` doesn't put one at the end. If you move the `\n` to the map:
```
const insertString = fshDefinition.mixins.map(m => `* insert ${m}\n`).join('');
```
That is the case where you do have an extra `\n` at the end. Unless I'm misinterpreting something.

Reply:
I think I just have not mastered the English language well enough yet.  That last bit was intended to mean this:
> ~~And~~ I don't think the final \n is necessary ~~, is it?~~. If ~~so~~ it is necessary, you could move the \n from the join to the map and just join on empty string.

So... My original suggested code does not put an `\n` at the very end, and I think that is OK -- but if it is not OK, then you can move that `\n` to the `map` in order to get the final `\n` back.

Reply:
Ahh now it all adds up. Gotcha. But you are correct, the final `\n` is really not necessary, so I will go with your original suggestion.